.github/workflows/board-automation.yaml

```diff
@@ -0,0 +1,17 @@
+name: Stripey
+
+on:
+  issues:
+    types:
+      - opened
+      - reopened
+jobs:
+  add_to_project:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: skeet70/default-project-board-action@v1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          repository: ${{ github.repository }}
+          issue: ${{ github.event.issue.number }}
+          project: 1
```

Comment:
nit: new line at end of file.

Reply:
ough.. sorry, yaml linter was sleeping on that...